package.json

```diff
@@ -5,39 +5,51 @@
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build --configuration=production --project=cps-ui-kit",
+    "build:documentation": "ng build --configuration=production --project=composition --base-href '/cps-shared-ui/'",
     "watch": "ng build cps-ui-kit --watch --configuration development",
     "test": "npm run cypress:run",
     "lint": "eslint \"**/*.ts\"",
     "cypress:open": "cypress open --component --browser chrome",
-    "cypress:run": "cypress run --component"
+    "cypress:run": "cypress run --component",
+    "generate-json-api": "node ./api-generator/api-generator.js",
+    "dev:ssr": "ng run composition:serve-ssr",
+    "serve:ssr": "node dist/composition/server/main.js",
+    "build:ssr": "ng build && ng run composition:server",
+    "prerender": "ng run composition:prerender"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^15.2.0",
-    "@angular/common": "^15.2.0",
-    "@angular/compiler": "^15.2.0",
-    "@angular/core": "^15.2.0",
-    "@angular/forms": "^15.2.0",
-    "@angular/platform-browser": "^15.2.0",
-    "@angular/platform-browser-dynamic": "^15.2.0",
-    "@angular/router": "^15.2.0",
+    "@angular/animations": "^17.1.0",
+    "@angular/common": "^17.1.0",
+    "@angular/compiler": "^17.1.0",
+    "@angular/core": "^17.1.0",
+    "@angular/forms": "^17.1.0",
+    "@angular/platform-browser": "^17.1.0",
+    "@angular/platform-browser-dynamic": "^17.1.0",
+    "@angular/platform-server": "^17.1.0",
+    "@angular/router": "^17.1.0",
+    "@angular/ssr": "^17.1.0",
     "@types/lodash-es": "^4.17.7",
+    "express": "^4.18.2",
     "lodash-es": "^4.17.21",
-    "primeng": "^15.4.1",
+    "primeng": "^17.3.2",
```

Comment:
pls update:
angular to 17.1.1
primeng to 17.4.0
@types/lodash-es to 4.17.12
rxjs to 7.8.1
tslib to 2.6.2
zone.js to 0.14.3
eslint to 8.52.0
prettier to 3.2.4
ng-packagr to 17.1.2
and so on :)

Reply:
Updated